forum/qa-plugin/report-reason/q2apro-flag-reasons-page.php

```diff
@@ -0,0 +1,209 @@
+<?php
+
+class q2apro_flag_reasons_page extends q2apro_flag_reasons_validation {
+
+    private $directory;
+    private $urlToRoot;
+
+    protected $CUSTOM_REPORT_REASON_ID = 0;
+    protected $_reportReasonRequestJSON = null;
```

Comment:
zapisywanie właściwości z _ na początku w PHP też jest raczej stary i wymierający

Reply:
Zapisem z "_" chciałem zakomunikować, żeby nie modyfikować wartości tego property. Czy jest jakiś lepszy sposób zapisu tego?

Reply:
Jak coś jest nie do modyfikacji to dajemy stałą. W innym przypadku to zwykła właściwość.

W PHP zapis z `_` na początku nazwy kiedyś powszechnie oznaczał prywatną właściwość, gdy jeszcze nie było językowo zaimplementowanych modyfikatorów widoczności. Od dawna te modyfikatory już jednak są, a zapis ten przestał być powszechny. Myślę więc, że większości skojarzy się z tym, że to jest prywatne i nikt nie pomyśli, że chodzi o to, aby nie modyfikować.

Reply:
Jest - protected const REPORT_REASON_REQUEST_JSON = null;